pkg/helpers/helpers.go

```diff
@@ -327,6 +328,31 @@ func updateManagedClusterStatus(client client.Client, managedClusterName string,
 	return true, nil
 }
 
+func SetImmediateImportCompleted(client client.Client, managedClusterName string) error {
+	managedCluster := &clusterv1.ManagedCluster{}
+	err := client.Get(context.TODO(), types.NamespacedName{Name: managedClusterName}, managedCluster)
+	if err != nil {
+		return err
+	}
+
+	ImmediateImport, ok := managedCluster.Annotations[apiconstants.AnnotationImmediateImport]
+	if !ok || ImmediateImport == apiconstants.AnnotationValueImmediateImportCompleted {
+		return nil
+	}
+
+	managedCluster = managedCluster.DeepCopy()
+	managedCluster.Annotations[apiconstants.AnnotationImmediateImport] = apiconstants.AnnotationValueImmediateImportCompleted
```

Comment:
does this mean that this annotation should not be set by an gitops tool.

Reply:
This annotation should not be set by a GitOps tool, as there's no valid reason for repeatedly triggering the import process of a managed cluster.